setup.py

```diff
@@ -19,8 +19,8 @@
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
       version='3.20',
       zip_safe=False,
-      test_suite="Chandra.test_Time",
-      packages=['Chandra'],
-      package_dir={'Chandra': 'Chandra'},
+      test_suite="Chandra.Time.test_Time",
+      packages=['Chandra', 'Chandra/Time'],
```

Comment:
I gave moving everything into `Chandra/Time` a shot, but it looked like it was going to make the 20 minute project a half-day project.  It also looked to me like moving `test_Time` into `tests` caused its own problem, so I left it in `Chandra/Time`.  If we have a driver for those moves I'll make it more of a project and will save and report more debugging info than 'it didn't work on the first go'.